lib/vtls/darwinssl.c

```diff
@@ -900,21 +903,43 @@ static OSStatus CopyIdentityWithLabel(char *label,
     keys[0] = kSecClass;
     values[1] = kCFBooleanTrue;    /* we want a reference */
     keys[1] = kSecReturnRef;
-    values[2] = kSecMatchLimitOne; /* one is enough, thanks */
+    values[2] = kSecMatchLimitAll; /* one is enough, thanks */
```

Comment:
if an unlimited number of results may be returned then the comment "one is enough" is wrong isn't it?